schema.sql/schema

```diff
@@ -0,0 +1,70 @@
+CREATE DATABASE catalog_of_things;
+
+-- Items
+CREATE TABLE items(
+    id VARCHAR(100) NOT NULL PRIMARY KEY,
+    genre_id VARCHAR(100),
+    author_id VARCHAR(100),
+    label_id VARCHAR(100),
+    source_id VARCHAR(100),
+    publish_date DATE NOT NULL,
+    archived BOOLEAN NOT NULL,
+    FOREIGN KEY (genre_id) REFERENCES genres (id),
+    FOREIGN KEY (author_id) REFERENCES authors (id),
+    FOREIGN KEY (label_id) REFERENCES labels (id),
+    FOREIGN KEY (source_id) REFERENCES sources (id),
+);
+
+
+CREATE TABLE Genre(
+  id      INTEGER GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  name    VARCHAR(100)
+);
+
+CREATE TABLE MusicAlbum(
```

Comment:
Hi @chaw-bot, Thanks for the review you're right in case ruby but this is a relational Schema or Database take some seconds to think about it and if you have any source or point to proof am wrong please share it on slack as most preferred as this PR might be close any moment from now.